exercism/bracket-push/bracket-push.js

```diff
@@ -0,0 +1,40 @@
+function bracket(str){
+var paren = ['(', ')']
+var curly = ['{','}']
+var square = ['[',']']
+var closings = [')', '}', ']']
+
+var arr = str.split('') //['{','}']
+var openingBracketsArr = []
```

Comment:
instead maybe call this openBrackets. It's a bit shorter and it is being assigned an empty array so it is rather explicit already.